core/src/main/scala/chisel3/PrintfImpl.scala

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+trait PrintfImpl {
```

Comment:
```suggestion
private[chisel3] trait PrintfImpl {
```
This needs to be package private